resources/tests.py

```diff
@@ -0,0 +1,23 @@
+from django.test import TestCase
+from .models import Resource
+
+
+class TestResourceModels(TestCase):
+
+    def test_Resource_created_correctly(self):
+
+        resource1 = Resource.objects.create(name = 'Forking', link = "https://www.youtube.com/watch?v=HbSjyU2vf6Y")
```

Comment:
I would move creating the Resource instances in the `setUp` method?
You can then reuse it also in your 2nd test case.

Reply:
I'm not sure if I've done that in a correct way, never used setUp() method before. Could you have a look, please?

Reply:
That looks right to me!